app/models/family.rb

```diff
@@ -143,8 +146,8 @@ class Family
 
```

Comment:
Add an index for eligibility_determination.subjects.eligibility_states.evidence_states.due_on